outreach/errors.go

```diff
@@ -10,4 +10,6 @@ var (
 	ErrNotObject     = errors.New("record is not an object")
 	ErrNotString     = errors.New("next is not a string")
 	ErrEmptyResponse = errors.New("empty response body")
+	ErrIdMustInt     = errors.New("provided id must be convertable to int")
+	ErrMustJSON      = errors.New("request data couldn't be decoded to json")
```

Comment:
There is `common.ErrNotJSON` but it is about response. This error could belong in `common`.
I think it is possible for other connectors to care about request being not json, should they parse it.